janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java

```diff
@@ -549,8 +549,7 @@ public void testCacheConcurrency() throws InterruptedException {
         System.out.println("Hits: " + (getEdgeCacheRetrievals()-getEdgeCacheMisses()));
         System.out.println("Misses: " + getEdgeCacheMisses());
         assertEquals(numReads, lookups.get());
-        final int numLookupPropertyConstraints = numV;
-        assertEquals(2 * numReads + numV + numLookupPropertyConstraints, getEdgeCacheRetrievals());
+        assertEquals(2 * numReads + numV + numV, getEdgeCacheRetrievals());
```

Comment:
This is a test so I think the additional variable is only there as an explanation of _why_ `numV` is added twice. Removing that would make me wonder why `numV` appears twice here.